sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb

```diff
@@ -372,7 +372,7 @@
     "\n",
     "When using AutoML for image tasks, you need to specify the model algorithms using the model_name parameter. You can either specify a single model or choose to sweep over multiple models. Please refer to the [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) for the list of supported model algorithms.\n",
     "\n",
```

Comment:
The v1 notebook also had these sections, could you please add them to this v2 notebook. "Convert annotation file from COCO to JSONL" and "Visualize bounding boxes"